utils/open_model_zoo/Security/instance_segmentation/0050/interp.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import cv2
+
+
+THRESHOLD = 0.5
+
+# See: https://github.com/opencv/open_model_zoo/blob/master/demos/python_demos/instance_segmentation_demo/main.py
+
+def segm_postprocess(box, raw_cls_mask, im_h, im_w):
+    # Add zero border to prevent upsampling artifacts on segment borders.
+    raw_cls_mask = np.pad(raw_cls_mask, ((1, 1), (1, 1)), 'constant', constant_values=0)
+    scale = int(raw_cls_mask.shape[0] / (raw_cls_mask.shape[0] - 2.0))
+    w_half = (box[2] - box[0]) * .5
+    h_half = (box[3] - box[1]) * .5
+    x_c = (box[2] + box[0]) * .5
+    y_c = (box[3] + box[1]) * .5
+    w_half *= scale
+    h_half *= scale
+    box_exp = np.zeros(box.shape)
+    box_exp[0] = x_c - w_half
+    box_exp[2] = x_c + w_half
+    box_exp[1] = y_c - h_half
+    box_exp[3] = y_c + h_half
+
+    extended_box = box_exp.astype(int)
+
+    w, h = np.maximum(extended_box[2:] - extended_box[:2] + 1, 1)
+    x0, y0 = np.clip(extended_box[:2], a_min=0, a_max=[im_w, im_h])
+    x1, y1 = np.clip(extended_box[2:] + 1, a_min=0, a_max=[im_w, im_h])
+
+    raw_cls_mask = cv2.resize(raw_cls_mask, (w, h)) > 0.5
+    mask = raw_cls_mask.astype(np.uint8)
+    # Put an object mask in an image mask.
+    im_mask = np.zeros((im_h, im_w), dtype=np.uint8)
```

Comment:
Note to self, this looks wrong. `im_w` and `im_h` appear flipped. need to investigate.